src/Sarif/Schemata/sarif-schema.json

```diff
@@ -48,18 +48,34 @@
   "definitions": {
 
     "address": {
-      "description": "The effective address of a reported issue.",
+      "description": "A physical or virtual address, or a range of addresses, in an 'addressable region' (memory or a binary file).",
       "additionalProperties": false,
       "type": "object",
       "properties": {
 
-        "baseAddress": {
-          "description": "A base address rendered as an integer value.",
+        "absoluteAddress": {
+          "description": "The address expressed as a byte offset from the start of the addressable region.",
+          "type": "integer",
+          "minimum": -1,
+          "default": -1
+
+        },
+
+        "relativeAddress": {
+          "description": "The address expressed as a byte offset from the absolute address of the top-most parent object.",
+          "type": "integer",
+          "minimum": -1,
+          "default": -1
```

Comment:
remove these!

also, these will need to be nullable types in the SDK.

Reply:
She has done so.

---
In reply to: [279567287](https://github.com/Microsoft/sarif-sdk/pull/1467#discussion_r279567287) [](ancestors = 279567287)